staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -456,6 +456,15 @@ func isDryRun(url *url.URL) bool {
 	return len(url.Query()["dryRun"]) != 0
 }
 
+// TODO: currently strict validation is set via
+// query param. Alternatively we could use a request header.
+// TODO: maybe we should check content-type here and error for protobuf (ie take the full req instead of just the URL?)
+// TODO: what do we want the query param to actually be (I went with ?validate=strict but am open to whatever)
```

Comment:
`?validation=strict`

Reply:
Changed to `?fieldValidation=Strict` based on [comment](https://github.com/kubernetes/enhancements/pull/2886#discussion_r702068352) from KEP. Param name not set in stone yet though, if you or anyone else has a better suggestion.